src/TLuaInterpreter.cpp

```diff
@@ -248,58 +250,88 @@ int TLuaInterpreter::Wait(lua_State* L)
     return 0;
 }
 
-// Documentation: ? - public function missing documentation in wiki
+// Documentation: PRIVATE function with no documentation in wiki.
+// dirToString wiil now catch and validate pretty much any string that could
```

Comment:
Here's a typo. Also the "now" qualifier is not useful here as we don't want to document past behaviour:
```suggestion
// dirToString will catch and validate pretty much any string that could
```

Reply:
:slightly_frowning_face: I think we will have to because it now means the functions respond to more possible exit designations ***as* normal *exits*** - which may hit some corner case Special exits in some hopefully obscure circumstances (e.g. if there was a Special exit called, for example `"East"` in the past it would be treated as a Special exit but now the custom exit line is regarded as a Normal exit in that direction).